### DISCUSSION TITLE
How to reference an attachment in a `documents.import` post?

### DISCUSSION THREAD

**Discussion by simonhorrobin-solidatus** [outline/outline#3474] [General] [answered]
If I create a document with an embedded svg file using the GUI then I can use `documents.export` to get hold of the markdown.

In it I get an api call which is structured as a bang prefixed link: `![](/api/attachments.redirect?id=<guid>`

If I POST this md back in then the bang prefixed link gets rendered literally - there's no substitution.  Is there any way to get it to render the attachment instead?

**Comment by tommoor** [comment, marked as answer]
Self-hosted or cloud? A lot of the import code paths changed recently. It sounds like you should probably be using `documents.create` endpoint though…

**Reply by simonhorrobin-solidatus** [reply]
`documents.create` works fine 😄
